framework/components/ecl-date-blocks/ecl-date-blocks.scss

```diff
@@ -0,0 +1,90 @@
+/*
+ * Date block
+ * @define date-block
+ */
+
+.ecl-date-block {
+  background-color: map-get($ecl-colors, 'grey-5');
+  border-bottom: 0.2em solid map-get($ecl-colors, 'yellow-100');
+  display: inline-flex;
+  margin: 0;
+  position: relative;
+
+  &::before {
+    border-left: 1.2em solid transparent;
+    border-top: 1.2em solid #fff;
+    content: '';
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+}
+
+.ecl-date-block__body {
+  align-items: center;
+  display: inline-flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: 5.55rem;
+  min-width: 5.55rem;
+}
+
+.ecl-date-block__week-day {
+  font-size: map-get($ecl-font-size, 'xs');
+  line-height: 1.2;
+  z-index: map-get($ecl-z-index, 'highlight');
```

Comment:
Well it was needed at some point, but I refactored my code, so I guess this is useless now.
Will remove it.